command/daemon/daemon.go

```diff
@@ -125,7 +127,13 @@ func (c *daemonCommand) run(*kingpin.ParseContext) error {
 		CertFile:            config.DefaultPoolSettings.CertFile,
 		KeyFile:             config.DefaultPoolSettings.KeyFile,
 	}
-	pools, err := cloudaws.ProcessPoolFile(c.poolFile, &defaultPoolSettings)
+	//pools, err := cloudaws.ProcessPoolFile(c.poolFile, &defaultPoolSettings)
```

Comment:
There is no need to comment these lines out. The `poolManager` supports multiple pools. Use something like:
```
poolsAWS, err := cloudaws.ProcessPoolFile(c.poolFile, &defaultPoolSettings)
err = poolManager.Add(poolsAWS...)

poolsGCP, err := google.ProcessPoolFile(c.googlePoolFile, &defaultPoolSettings)
err = poolManager.Add(poolsGCP...)
```

We'll refactor it later when we switch to using a single pool file solution.